www/front_src/src/Navigation/Sidebar/Menu/CollapsableItems.tsx

```diff
@@ -0,0 +1,367 @@
+    },
+    '& .MuiSvgIcon-root': {
+      color: theme.palette.background.paper,
+    },
+    '&:hover': {
+      backgroundColor: theme.palette.primary.main,
+    },
+
+    backgroundColor: theme.palette.primary.main,
+  },
+  label: {
+    fontWeight: 'bold',
+  },
+  root: {
+    '& .MuiListItemIcon-root': {
+      minWidth: theme.spacing(2.25),
+      padding: theme.spacing(0, 0.25, 0, 0.75),
+    },
+    '& .MuiTypography-root': {
+      color: theme.palette.text.primary,
+      fontSize: theme.typography.caption,
+    },
+    outline: `solid  ${theme.palette.divider} ${0.1}px`,
+  },
+  subHeader: {
+    color: theme.palette.text.secondary,
+    fontSize: theme.typography.caption.fontSize,
+    fontWeight: 'bold',
+    lineHeight: theme.spacing(2.9),
+    textAlign: 'center',
+  },
+  toggled: {
+    '&::-webkit-scrollbar': {
+      width: theme.spacing(1),
+    },
+    '&::-webkit-scrollbar-corner': {
+      backgroundColor: theme.palette.background.default,
+    },
+    '&::-webkit-scrollbar-thumb': {
+      backgroundColor: theme.palette.action.disabled,
+    },
+    '&::-webkit-scrollbar-track': {
+      border: `solid ${theme.palette.action.hover} 0.5px`,
+    },
+    backgroundColor: theme.palette.background.default,
+    left: ({ currentWidth }: StyleProps): string => theme.spacing(currentWidth),
+    maxHeight: ({ collapseScrollMaxHeight }: StyleProps): string =>
+      collapseScrollMaxHeight
+        ? theme.spacing(collapseScrollMaxHeight)
+        : theme.spacing(50),
+    maxWidth: ({ collapseScrollMaxWidth }: StyleProps): string =>
+      collapseScrollMaxWidth
+        ? theme.spacing(collapseScrollMaxWidth)
+        : theme.spacing(collapseWidth),
+    overflow: 'auto',
+    position: 'fixed',
+    top: ({ currentTop }: StyleProps): number | undefined => currentTop,
+    width: theme.spacing(collapseWidth),
+    zIndex: theme.zIndex.mobileStepper,
+  },
+}));
+
+const CollapsableItems = ({
+  data,
+  isCollapsed,
+  isSubHeader,
+  currentTop,
+  currentWidth,
+  onClick,
+  onLeave,
+  level,
+  collapseScrollMaxHeight,
+  collapseScrollMaxWidth,
+  setCollapseScrollMaxWidth,
+  setCollapseScrollMaxHeight,
+}: Props): JSX.Element => {
+  const classes = useStyles({
+    collapseScrollMaxHeight,
+    collapseScrollMaxWidth,
+    currentTop,
+    currentWidth,
+  });
+  const [hoveredIndex, setHoveredIndex] = useState<number | null>(null);
+  const [itemTop, setItemTop] = useState<number>();
+  const [nestedScrollCollapsMaxHeight, setNestedScrollCollapsMaxHeight] =
+    useState<undefined | number>(undefined);
+  const [nestedScrollCollapsMaxWidth, setNestedScrollCollapsMaxWidth] =
+    useState<undefined | number>(undefined);
+  const collapsRef = React.useRef<HTMLElement | null>(null);
+  const [hoveredNavigationItems, setHoveredNavigationItems] = useAtom(
+    hoveredNavigationItemsAtom,
+  );
+  const selectedNavigationItems = useAtomValue(selectedNavigationItemsAtom);
+
+  const levelName = `level_${level}_Navigated`;
```

Comment:
```suggestion
  const levelName = `level_${level}`;
```
You can rid of `_Navigated`